### PR TITLE
testing_semantics

### PR DESCRIPTION
**Pull request by rizel-botany**

### CODE REVIEW

**Comment by rizel-botany** [comment]
@norman-promise-neverland you are so close

**Comment by norman-promise-neverland** [comment]
don't want to hurt you @rizel-botany

File: README.md [hunk 1]
```diff
@@ -6,5 +6,5 @@ yo
 
 :smile: 
 ok
-:satisfied:
+:satisfied:dasdfadsfsda
```
Comment (norman-promise-neverland) [inline README.md:9]: paying @rizel-botany to do this

**Comment by norman-promise-neverland** [comment]
graduating @rizel-botany

**Comment by norman-promise-neverland** [comment]
pizza monday @rizel-botany

**Comment by norman-promise-neverland** [comment]
quality time @rizel-botany

**Comment by norman-promise-neverland** [comment]
watching all american @rizel-botany

**Review by norman-promise-neverland — APPROVED** [review]

**Pull request merged**